2020/06.md

```diff
@@ -84,6 +84,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 2     | 45m     | [Temporal](https://github.com/tc39/proposal-temporal) update (slides TBD) | Philip Chimento (and others TBD) |
     |   | 2     | 60m     | [Function Implementation Hiding](https://github.com/tc39/proposal-function-implementation-hiding) for stage 3 ([slides](https://docs.google.com/presentation/d/1zCACnOkueOZHruXv2UzUvKIUA6L7w7W6S2N4bdzwwkM/edit)) ([spec text](https://github.com/tc39/ecma262/pull/1739)) | Michael Ficarra |
     |   | 2     | 60m     | [Realms, Stage 2 updates](https://github.com/tc39/proposal-realms) (slides TBD) | Caridy Patiño, Leo Balter |
+    |   | 2     | 30m     | [Intl.Segmenter](https://github.com/tc39/proposal-intl-segmenter) for Stage 3 ([slides](https://docs.google.com/presentation/d/1Pe9eVhgK93cgB3KCufTQvzqCjIYj3RRxJaOeNIbWN_A/edit?usp=sharing), [spec text](https://tc39.es/proposal-intl-segmenter/)) | Richard Gibson |
```

Comment:
Unfortunately, deadline for advancement eligibility has passed on May 23 :(

https://github.com/tc39/agendas/blob/master/2020/06.md#agenda-topic-rules

Reply:
I'll add the hourglass (and also, properly sort)